prow/config.yaml

```diff
@@ -257,6 +257,19 @@ tide:
     - do-not-merge/invalid-owners-file
     - do-not-merge/work-in-progress
     - needs-ok-to-test
+  - repos:
+    - kubernetes-incubator/service-catalog
+    labels:
+    - lgtm
+    - approved
+    - "cncf-cla: yes"
+    - LGTM1
+    - LGTM2
+    missingLabels:
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - needs-ok-to-test
+    - needs-rebase
```

Comment:
How do I get this label automatically applied?

This is described as an `external plugin`. Is it available for me to enable?

Reply:
Okay, I added what I think is the appropriate thing to plugins.yaml